sites/all/themes/orwell/sass/component/_article.scss

```diff
@@ -170,46 +188,51 @@ $teaser-height--medium: 400px;
   &__covers {
     position: relative;
     height: 100vw;
-    max-height: 360px;
-    max-width: 400px;
+    max-height: 220px;
+    max-width: 300px;
     margin: auto;
 
-    img:nth-child(1) {
-      position: absolute;
-      width: 59%;
-      height: 212px;
-      max-width: 17rem;
-      left: 2%;
-      object-fit: cover;
-      transform: translate(0%, 0%) rotate(0deg) scale(1);
-      z-index: 2;
-      box-shadow: 11px -1px 28px -13px rgba(0,0,0,0.75);
-
-      @include grid-media($xsmall) {
-        height: 335px;
-      }
-
-      @include grid-media($medium) {
-        transform: translate(59%, 0%) rotate(0deg) scale(1);
-        box-shadow: -11px -1px 28px -13px rgba(0,0,0,0.75);
-      }
+    @include grid-media($medium) {
+      max-height: 340px;
+      max-width: 400px;
     }
 
-    img:nth-child(2) {
+    // Style covers in different sizes and make them overlap a bit.
+    img {
       position: absolute;
-      height: 136px;
-      width: 39%;
-      max-width: 12.5rem;
+      width: auto;
       object-fit: cover;
-      transform: translate(150%, 39%) rotate(0deg) scale(1);
-      z-index: 1;
-
-      @include grid-media($xsmall) {
-        height: 229px;
+      max-height: 100%;
+
+      &:nth-child(1) {
```

Comment:
Could do with some more comments in the styling - Eg. I have absolutely no idea why the first child needs its z-index adjusted :)